tests/Fixtures/ListFilesTest/.php-cs-fixer.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+$finder = PhpCsFixer\Finder::create()
+    ->in([
+        __DIR__.'/needs-fixing/'
+    ])
+    ->exclude([
+        __DIR__.'/excluded/'
+    ])
+;
+
+return PhpCsFixer\Config::create()
```

Comment:
please fix the deprecation travis is reporting: https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/5390/checks?check_run_id=2351718634#step:11:523